sdk/identity/identity/integration/AzureKubernetes/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+ARG NODE_VERSION=20
+
+# docker can't tell when the repo has changed and will therefore cache this layer
+# internal users should provide MCR registry to build via 'docker build . --build-arg REGISTRY="mcr.microsoft.com/mirror/docker/library/"'
+# public OSS users should simply leave this argument blank or ignore its presence entirely
+ARG REGISTRY=""
+
+FROM ${REGISTRY}node:${NODE_VERSION}-alpine as repo
+RUN apk --no-cache add git
+RUN git clone https://github.com/azure/azure-sdk-for-js --single-branch --branch main --depth 1 /azure-sdk-for-js
```

Comment:
Let me check this part on the live test and get back to you.

Reply:
Actually, for AKS, for testing and merging we'll need to have different branch names/ repo names in the dockerfile.
When I am testing my PR, I should be using my own repo name and branch name but for the live test pipeline, this should be pointing to main.

Should I put a comment in there? What would you suggest?

Reply:
Instead of cloning the repo, you can COPY the files you need over to the container. It'll be faster, will not depend on github, and will allow you to test the code in the PR (instead of main). Let me play around with your branch this morning and share an example of how to do that. Meanwhile, what are other languages doing?

Reply:
This shows an example of how to do that (copy the files over) https://github.com/KarishmaGhiya/azure-sdk-for-js/pull/8 - let me know if you get stuck and we can pair!

Reply:
Thanks @maorleger I'll probably do this in a follow up PR,  when working on Kubernetes. This is useful.